compiler/src/typed/oprint.re

```diff
@@ -696,10 +723,11 @@ and print_out_type_decl = (kwd, ppf, td) => {
     | _ =>
       fprintf(
         ppf,
-        "@[(@[%a)@]@ %s@]",
+        // TODO: Fix the box, `<` and `>` are special characters in a box
```

Comment:
Shouldn't be applicable here, but, FWIW, the lt/gt symbols become special after an open box (maybe that caused an issue during development?); see the first bullet point [here](https://ocaml.org/releases/4.10/htmlman/libref/Format.html#VALfprintf).

Reply:
Ah yeah sorry, should have worded this better.

Reply:
Yeah, what @peblair said is what happened, the outer box was causing the `<` and `>` to become special characters inside the box and I don't actually understand how to escape them.

Reply:
@phated Right, but I meant that maybe you ran into that during development and it's no longer an issue (since the open box is gone now)? Or is it still a problem?

Reply:
The TODO is about restoring the outer box that I removed. Do you think that the outer box doesn't need to be there?

Reply:
I thought it needed to be there in case the type params needed to break a line, like:
```
enum SomeReallyReallyReallyReallyReallyReallyReallyLongName<
    myParam1, myParam2
>

Reply:
```suggestion
        // TODO: This no longer has an outer box, nor break hints, because `<` and `>` are special characters in a box. If this causes issues with line-wrapping, we should fix in the future.
```
@ospencer @peblair how about this comment?